internal/linker/test/issue_1813/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@npm//jest-cli:index.bzl", "jest_test")
+
+jest_test(
+    name = "test",
+    data = [
+        "index.test.js",
+        "jest.config.js",
+        "@npm//jest",
+        "@npm//jest-websocket-mock",
+        "@npm//mock-socket",
+    ],
+    templated_args = [
+        "--no-cache",
+        "--no-watchman",
+        "--ci",
+        # On Windows, spawing workers is broken
+        "--runInBand",
+        "--config",
+        "$$(rlocation $(rootpath jest.config.js))",
```

Comment:
random note: any time we write this under the `/examples` directory we should comment about it

Reply:
It would be rad if this were `args=[".dist/bin/jest.config.js"])` and we fixed it up later in the toolchain